### PR TITLE
Updates for Change Log and deprecated config reference items

### PR DESCRIPTION
**Pull request by leslietilling**
## Purpose of this pull request

This pull request (PR) updates the change log with Services topic and config references for the April pre-release beta publish. Also adds matching deprecation notes to for the payment methods in the associated configuration reference topics. 

## Affected documentation pages

- docs.magento.com/m2/ee/user_guide/configuration/sales/authorize-net-direct-post.html
- docs.magento.com/m2/ee/user_guide/configuration/sales/authorize-net.html
- docs.magento.com/m2/ee/user_guide/configuration/sales/cybersource.html
- docs.magento.com/m2/ee/user_guide/configuration/sales/eway.html
- docs.magento.com/m2/ee/user_guide/configuration/sales/fraud-protection.html
- docs.magento.com/m2/ee/user_guide/configuration/sales/payment-methods-other.html
- docs.magento.com/m2/ee/user_guide/configuration/sales/worldpay.html
- docs.magento.com/m2/ee/user_guide/payment/authorize-net-direct-post.html
- docs.magento.com/m2/ee/user_guide/payment/authorize-net.html
- docs.magento.com/m2/ee/user_guide/payment/cybersource.html
- docs.magento.com/m2/ee/user_guide/payment/eway.html
- docs.magento.com/m2/ee/user_guide/magento/change-log.html

## Affected Magento editions

- [x] Open Source
- [x] Commerce
- [x] B2B

### CODE REVIEW

File: src/configuration/sales/authorize-net-direct-post.md [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
+title: Authorize.Net Direct Post - Deprecated
 ---
 
 Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Sales]({% link configuration/sales.md %}) > [Payment Methods]({% link configuration/sales/payment-methods.md %}) > Authorize.Net Direct Post (Deprecated)
 
 {:.bs-callout-warning}
-Authorize.Net has deprecated the Authorize.Net Direct Post payment method. You must configure the Authorize.Net payment method as the replacement method. Refer to [Authorize.Net]({% link configuration/sales/authorize-net.md %}) for more information.
+**Deprecation Notice** <br/>
+Due to the Payment Service Directive [PSD2]({% link stores/compliance-payment-services-directive.md %}) and the continued evolution of many APIs, this payment integration is at risk of becoming outdated and no longer security compliant in the future. Additionally, Authorize.Net has deprecated the Authorize.Net Direct Post payment method. We are recommending that you disable it in your Magento configuration and transition to the Authorize.Net [Magento Marketplace extension](https://marketplace.magento.com/catalogsearch/result/?q=authorize.net){:target="_blank"}.<br/><br/>
```
Comment (hickskenh) [inline src/configuration/sales/authorize-net-direct-post.md:9]:
```suggestion
Due to the Payment Service Directive [PSD2]({% link stores/compliance-payment-services-directive.md %}) and the continued evolution of many APIs, this payment integration is at risk of becoming outdated and no longer security compliant in the future. Additionally, Authorize.Net has deprecated the Authorize.Net Direct Post payment method. We recommend that you disable it in your Magento configuration and transition to the Authorize.Net [Magento Marketplace extension](https://marketplace.magento.com/catalogsearch/result/?q=authorize.net){:target="_blank"}.<br/><br/>
```

File: src/configuration/sales/authorize-net.md [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
+**Deprecation Notice** <br/>
+Due to the Payment Service Directive [PSD2]({% link stores/compliance-payment-services-directive.md %}) and the continued evolution of many APIs, this payment integration is at risk of becoming outdated and no longer security compliant in the future. For this reason, it is now deprecated and we are recommending that you disable it in your Magento configuration and transition to the corresponding [Magento Marketplace extension](https://marketplace.magento.com/catalogsearch/result/?q=authorize.net){:target="_blank"}.<br/><br/>
```
Comment (hickskenh) [inline src/configuration/sales/authorize-net.md:9]:
```suggestion
Due to the Payment Service Directive [PSD2]({% link stores/compliance-payment-services-directive.md %}) and the continued evolution of many APIs, this payment integration is at risk of becoming outdated and no longer security compliant in the future. For this reason, it is now deprecated, and we recommend that you disable it in your Magento configuration and transition to the corresponding [Magento Marketplace extension](https://marketplace.magento.com/catalogsearch/result/?q=authorize.net){:target="_blank"}.<br/><br/>
```

File: src/configuration/sales/cybersource.md [hunk 3]
```diff
@@ -1,13 +1,15 @@
...
+**Deprecation Notice** <br/>
+Due to the Payment Service Directive [PSD2]({% link stores/compliance-payment-services-directive.md %}) and the continued evolution of many APIs, this payment integration is at risk of becoming outdated and no longer security compliant in the future. For this reason, it is now deprecated and we are recommending that you disable it in your Magento configuration and transition to the corresponding [Magento Marketplace extension](https://marketplace.magento.com/catalogsearch/result/?q=cybersource#q=cybersource&idx=m2_cloud_prod_default_products&p=0&nR%5Bvisibility_search%5D%5B%3D%5D%5B0%5D=1){:target="_blank"}.<br/><br/>
```
Comment (hickskenh) [inline src/configuration/sales/cybersource.md:10]:
```suggestion
Due to the Payment Service Directive [PSD2]({% link stores/compliance-payment-services-directive.md %}) and the continued evolution of many APIs, this payment integration is at risk of becoming outdated and no longer security compliant in the future. For this reason, it is now deprecated, and we recommend that you disable it in your Magento configuration and transition to the corresponding [Magento Marketplace extension](https://marketplace.magento.com/catalogsearch/result/?q=cybersource#q=cybersource&idx=m2_cloud_prod_default_products&p=0&nR%5Bvisibility_search%5D%5B%3D%5D%5B0%5D=1){:target="_blank"}.<br/><br/>
```

File: src/configuration/sales/eway.md [hunk 4]
```diff
@@ -1,13 +1,15 @@
 ---
 conditions: Default.EE-B2B
-title: eWAY (Deprecated)
+title: eWAY - Deprecated
 ---
 
 Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Sales]({% link configuration/sales.md %}) > [Payment Methods]({% link configuration/sales/payment-methods.md %}) > eWAY (Deprecated)
```
Comment (hickskenh) [inline src/configuration/sales/eway.md:6]: change () in path to - Deprecated

File: src/configuration/sales/eway.md [hunk 5]
```diff
@@ -1,13 +1,15 @@
...
 
 {:.bs-callout-warning}
-**Payment Services Directive Requirements:** <br/>
-As of September 14, 2019, European banks might decline payments that do not meet [PSD2]({% link stores/compliance-payment-services-directive.md %}) requirements. To comply with PSD2, install and configure the official eWAY payment integration extension from [Magento Marketplace](https://marketplace.magento.com/catalogsearch/result/?q=eway){:target="_blank"}.
+**Deprecation Notice** <br/>
+Due to the Payment Service Directive [PSD2]({% link stores/compliance-payment-services-directive.md %}) and the continued evolution of many APIs, this payment integration is at risk of becoming outdated and no longer security compliant in the future. For this reason, it is now deprecated and we are recommending that you disable it in your Magento configuration and transition to the corresponding [Magento Marketplace extension](https://marketplace.magento.com/catalogsearch/result/?q=eway){:target="_blank"}.<br/><br/>
```
Comment (hickskenh) [inline src/configuration/sales/eway.md:10]:
```suggestion
Due to the Payment Service Directive [PSD2]({% link stores/compliance-payment-services-directive.md %}) and the continued evolution of many APIs, this payment integration is at risk of becoming outdated and no longer security compliant in the future. For this reason, it is now deprecated, and we recommend that you disable it in your Magento configuration and transition to the corresponding [Magento Marketplace extension](https://marketplace.magento.com/catalogsearch/result/?q=eway){:target="_blank"}.<br/><br/>
```

File: src/configuration/sales/fraud-protection.md [hunk 6]
```diff
@@ -6,6 +6,10 @@ Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Sale
 
 ## Signifyd
 
+{:.bs-callout-warning}
+**Deprecation Notice** <br/>
+Due to the continued evolution of many APIs, the Signifyd Guaranteed Fraud Protection fraud protection integration is at risk of becoming outdated and no longer security compliant in the future. For this reason, it is now deprecated and we are recommending that you disable it in your Magento configuration and transition to the [Signifyd Fraud & Chargeback Protection extension](https://marketplace.magento.com/signifyd-module-connect.html){:target="_blank"} that is available on Magento Marketplace.
```
Comment (hickskenh) [inline src/configuration/sales/fraud-protection.md:11]:
```suggestion
Due to the continued evolution of many APIs, the Signifyd Guaranteed Fraud Protection fraud protection integration is at risk of becoming outdated and no longer security compliant in the future. For this reason, it is now deprecated, and we recommend that you disable it in your Magento configuration and transition to the [Signifyd Fraud & Chargeback Protection extension](https://marketplace.magento.com/signifyd-module-connect.html){:target="_blank"} that is available on Magento Marketplace.
```

File: src/configuration/sales/worldpay.md [hunk 7]
```diff
@@ -1,13 +1,15 @@
...
 ---
 
 Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Sales]({% link configuration/sales.md %}) > [Payment Methods]({% link configuration/sales/payment-methods.md %}) > Worldpay
 
 {:.bs-callout-warning}
-**Payment Services Directive Requirements:** <br/>
-As of September 14, 2019, European banks might decline payments that do not meet [PSD2]({% link stores/compliance-payment-services-directive.md %}) requirements. To comply with PSD2, contact your Worldpay service provider and ask for their recommendation.
+**Deprecation Notice** <br/>
+Due to the Payment Service Directive [PSD2]({% link stores/compliance-payment-services-directive.md %}) and the continued evolution of many APIs, this payment integration is at risk of becoming outdated and no longer security compliant in the future. For this reason, it is now deprecated and we are recommending that you disable it in your Magento configuration and contact your Worldpay service provider and ask for their recommendation.<br/><br/>
+**This integration will be removed from the Magento 2.4.0 release and has been deprecated from current versions of 2.3.**<br/><br/>
```
Comment (hickskenh) [inline src/configuration/sales/worldpay.md:11]: this isn't bolded in other topics.  also, current or all versions?

File: src/configuration/sales/worldpay.md [hunk 8]
```diff
@@ -1,13 +1,15 @@
...
+title: Worldpay - Deprecated
 ---
 
 Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Sales]({% link configuration/sales.md %}) > [Payment Methods]({% link configuration/sales/payment-methods.md %}) > Worldpay
 
 {:.bs-callout-warning}
-**Payment Services Directive Requirements:** <br/>
-As of September 14, 2019, European banks might decline payments that do not meet [PSD2]({% link stores/compliance-payment-services-directive.md %}) requirements. To comply with PSD2, contact your Worldpay service provider and ask for their recommendation.
+**Deprecation Notice** <br/>
+Due to the Payment Service Directive [PSD2]({% link stores/compliance-payment-services-directive.md %}) and the continued evolution of many APIs, this payment integration is at risk of becoming outdated and no longer security compliant in the future. For this reason, it is now deprecated and we are recommending that you disable it in your Magento configuration and contact your Worldpay service provider and ask for their recommendation.<br/><br/>
```
Comment (hickskenh) [inline src/configuration/sales/worldpay.md:10]:
```suggestion
Due to the Payment Service Directive [PSD2]({% link stores/compliance-payment-services-directive.md %}) and the continued evolution of many APIs, this payment integration is at risk of becoming outdated and no longer security compliant in the future. For this reason, it is now deprecated, and we recommend that you disable it in your Magento configuration and contact your Worldpay service provider and ask for their recommendation.<br/><br/>
```

File: src/payment/authorize-net.md [hunk 9]
```diff
@@ -6,7 +6,7 @@ tag: payments-deprecated
 {:.bs-callout-warning}
 **Deprecation Notice** <br/>
 Due to the Payment Service Directive [PSD2]({% link stores/compliance-payment-services-directive.md %}) and the continued evolution of many APIs, this payment integration is at risk of becoming outdated and no longer security compliant in the future. For this reason, it is now deprecated and we are recommending that you disable it in your Magento configuration and transition to the corresponding [Magento Marketplace extension](https://marketplace.magento.com/catalogsearch/result/?q=authorize.net){:target="_blank"}.<br/><br/>
-**This integration will be removed from the Magento 2.4.0 release and has been deprecated from versions of 2.3.**<br/><br/>
+**This integration will be removed from the Magento 2.4.0 release and has been deprecated from current versions of 2.3.**<br/><br/>
```
Comment (hickskenh) [inline src/payment/authorize-net.md:9]: current or all?

File: src/payment/cybersource.md [hunk 10]
```diff
@@ -6,7 +6,7 @@ title: CyberSource - Deprecated
 {:.bs-callout-warning}
 **Deprecation Notice** <br/>
 Due to the Payment Service Directive [PSD2]({% link stores/compliance-payment-services-directive.md %}) and the continued evolution of many APIs, this payment integration is at risk of becoming outdated and no longer security compliant in the future. For this reason, it is now deprecated and we are recommending that you disable it in your Magento configuration and transition to the corresponding [Magento Marketplace extension](https://marketplace.magento.com/catalogsearch/result/?q=cybersource#q=cybersource&idx=m2_cloud_prod_default_products&p=0&nR%5Bvisibility_search%5D%5B%3D%5D%5B0%5D=1){:target="_blank"}.<br/><br/>
```
Comment (hickskenh) [inline src/payment/cybersource.md:8]:
```suggestion
Due to the Payment Service Directive [PSD2]({% link stores/compliance-payment-services-directive.md %}) and the continued evolution of many APIs, this payment integration is at risk of becoming outdated and no longer security compliant in the future. For this reason, it is now deprecated, and we recommend that you disable it in your Magento configuration and transition to the corresponding [Magento Marketplace extension](https://marketplace.magento.com/catalogsearch/result/?q=cybersource#q=cybersource&idx=m2_cloud_prod_default_products&p=0&nR%5Bvisibility_search%5D%5B%3D%5D%5B0%5D=1){:target="_blank"}.<br/><br/>
```

File: src/payment/cybersource.md [hunk 11]
```diff
@@ -6,7 +6,7 @@ title: CyberSource - Deprecated
...
-**This integration will be removed from the Magento 2.4.0 release and has been deprecated from versions of 2.3.**<br/><br/>
+**This integration will be removed from the Magento 2.4.0 release and has been deprecated from current versions of 2.3.**<br/><br/>
```
Comment (hickskenh) [inline src/payment/cybersource.md:9]: current or all?

File: src/payment/eway.md [hunk 12]
```diff
@@ -6,7 +6,7 @@ title: eWAY - Deprecated
 {:.bs-callout-warning}
 **Deprecation Notice** <br/>
 Due to the Payment Service Directive [PSD2]({% link stores/compliance-payment-services-directive.md %}) and the continued evolution of many APIs, this payment integration is at risk of becoming outdated and no longer security compliant in the future. For this reason, it is now deprecated and we are recommending that you disable it in your Magento configuration and transition to the corresponding [Magento Marketplace extension](https://marketplace.magento.com/catalogsearch/result/?q=eway){:target="_blank"}.<br/><br/>
```
Comment (hickskenh) [inline src/payment/eway.md:8]:
```suggestion
Due to the Payment Service Directive [PSD2]({% link stores/compliance-payment-services-directive.md %}) and the continued evolution of many APIs, this payment integration is at risk of becoming outdated and no longer security compliant in the future. For this reason, it is now deprecated, and we recommend that you disable it in your Magento configuration and transition to the corresponding [Magento Marketplace extension](https://marketplace.magento.com/catalogsearch/result/?q=eway){:target="_blank"}.<br/><br/>
```

File: src/payment/eway.md [hunk 13]
```diff
@@ -6,7 +6,7 @@ title: eWAY - Deprecated
...
-<b>This integration will be removed from the Magento 2.4.0 release and has been deprecated from versions of 2.3.</b><br/><br/>
+**This integration will be removed from the Magento 2.4.0 release and has been deprecated from current versions of 2.3.**<br/><br/>
```
Comment (hickskenh) [inline src/payment/eway.md:9]: current or all?

File: src/configuration/sales/authorize-net-direct-post.md [hunk 14]
```diff
@@ -1,11 +1,14 @@
...
+**This integration will be removed from the Magento 2.4.0 release and has been deprecated from current versions of 2.3.**<br/><br/>
```
[the remark this answers is not in the document]
Comment (leslietilling) [reply src/configuration/sales/authorize-net-direct-post.md:10]: After discussion with Misha, "current" is the best. We have the link to the DevBlog if they need the specific version information.

File: src/configuration/sales/authorize-net-direct-post.md [hunk 15]
```diff
@@ -1,11 +1,14 @@
...
+For details about making a secure transition from deprecated payment integrations, see our [DevBlog](https://community.magento.com/t5/Magento-DevBlog/Deprecation-of-Magento-core-payment-integrations/ba-p/426445){:target="_blank"}.
 
 ![]({% link images/images/config-sales-payment-methods-authorize-net-direct-post.png %}){: .zoom}
 _Authorize.Net Direct Post (Deprecated)_
```
[the remark this answers is not in the document]
Comment (leslietilling) [reply src/configuration/sales/authorize-net-direct-post.md:14]: I think in the caption it is fine

File: src/configuration/sales/authorize-net.md [hunk 16]
```diff
@@ -1,12 +1,14 @@
...
+**This integration will be removed from the Magento 2.4.0 release and has been deprecated from current versions of 2.3.**<br/><br/>
```
[the remark this answers is not in the document]
Comment (leslietilling) [reply src/configuration/sales/authorize-net.md:10]: Same as above

File: src/configuration/sales/cybersource.md [hunk 17]
```diff
@@ -1,13 +1,15 @@
...
+**This integration will be removed from the Magento 2.4.0 release and has been deprecated from current versions of 2.3.**<br/><br/>
+For details about making a secure transition from deprecated payment integrations, see our [DevBlog](https://community.magento.com/t5/Magento-DevBlog/Deprecation-of-Magento-core-payment-integrations/ba-p/426445){:target="_blank"}.
 
 ![]({% link images/images-ee/config-sales-payment-methods-cybersource.png %}){: .zoom}
 [_CyberSource_]({% link payment/cybersource.md %})
```
[the remark this answers is not in the document]
Comment (leslietilling) [reply src/configuration/sales/cybersource.md:15]: Same as above

File: src/configuration/sales/cybersource.md [hunk 18]
```diff
@@ -1,13 +1,15 @@
...
+**Deprecation Notice** <br/>
+Due to the Payment Service Directive [PSD2]({% link stores/compliance-payment-services-directive.md %}) and the continued evolution of many APIs, this payment integration is at risk of becoming outdated and no longer security compliant in the future. For this reason, it is now deprecated and we are recommending that you disable it in your Magento configuration and transition to the corresponding [Magento Marketplace extension](https://marketplace.magento.com/catalogsearch/result/?q=cybersource#q=cybersource&idx=m2_cloud_prod_default_products&p=0&nR%5Bvisibility_search%5D%5B%3D%5D%5B0%5D=1){:target="_blank"}.<br/><br/>
+**This integration will be removed from the Magento 2.4.0 release and has been deprecated from current versions of 2.3.**<br/><br/>
```
[the remark this answers is not in the document]
Comment (leslietilling) [reply src/configuration/sales/cybersource.md:11]: Same as above

File: src/payment/authorize-net.md (hunk 9 above)
Comment (leslietilling) [reply src/payment/authorize-net.md:9]: Same as above

File: src/configuration/sales/eway.md [hunk 19]
```diff
@@ -1,13 +1,15 @@
...
+**This integration will be removed from the Magento 2.4.0 release and has been deprecated from current versions of 2.3.**<br/><br/>
+For details about making a secure transition from deprecated payment integrations, see our [DevBlog](https://community.magento.com/t5/Magento-DevBlog/bg-p/devblog){:target="_blank"}.
 
 ![]({% link images/images-ee/config-sales-payment-methods-eway.png %}){: .zoom}
 [_eWAY_]({% link payment/eway.md %})
```
[the remark this answers is not in the document]
Comment (leslietilling) [reply src/configuration/sales/eway.md:15]: Same as above

File: src/configuration/sales/eway.md [hunk 20]
```diff
@@ -1,13 +1,15 @@
...
 
 {:.bs-callout-warning}
-**Payment Services Directive Requirements:** <br/>
-As of September 14, 2019, European banks might decline payments that do not meet [PSD2]({% link stores/compliance-payment-services-directive.md %}) requirements. To comply with PSD2, install and configure the official eWAY payment integration extension from [Magento Marketplace](https://marketplace.magento.com/catalogsearch/result/?q=eway){:target="_blank"}.
+**Deprecation Notice** <br/>
+Due to the Payment Service Directive [PSD2]({% link stores/compliance-payment-services-directive.md %}) and the continued evolution of many APIs, this payment integration is at risk of becoming outdated and no longer security compliant in the future. For this reason, it is now deprecated and we are recommending that you disable it in your Magento configuration and transition to the corresponding [Magento Marketplace extension](https://marketplace.magento.com/catalogsearch/result/?q=eway){:target="_blank"}.<br/><br/>
+**This integration will be removed from the Magento 2.4.0 release and has been deprecated from current versions of 2.3.**<br/><br/>
```
[the remark this answers is not in the document]
Comment (leslietilling) [reply src/configuration/sales/eway.md:11]: Same as above

File: src/configuration/sales/worldpay.md [hunk 21]
```diff
@@ -1,13 +1,15 @@
...
+**This integration will be removed from the Magento 2.4.0 release and has been deprecated from current versions of 2.3.**<br/><br/>
+For details about making a secure transition from deprecated payment integrations, see our [DevBlog](https://community.magento.com/t5/Magento-DevBlog/Deprecation-of-Magento-core-payment-integrations/ba-p/426445){:target="_blank"}.
 
 ![]({% link images/images-ee/config-sales-payment-methods-worldpay.png %}){: .zoom}
 [_Worldpay_]({% link payment/worldpay.md %})
```
[the remark this answers is not in the document]
Comment (leslietilling) [reply src/configuration/sales/worldpay.md:15]: Same as above

File: src/payment/eway.md (hunk 13 above)
Comment (leslietilling) [reply src/payment/eway.md:9]: Same as above

File: src/payment/cybersource.md (hunk 11 above)
Comment (leslietilling) [reply src/payment/cybersource.md:9]: Same as above

File: src/configuration/sales/worldpay.md (hunk 7 above)
Comment (leslietilling) [reply src/configuration/sales/worldpay.md:11]: They are bolded. But doesn't always display that way in GitHub review UI.

File: src/configuration/sales/fraud-protection.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (leslietilling) [reply src/configuration/sales/fraud-protection.md]: fixed

**Pull request merged**